### ISSUE TITLE
Automated smoke test for caikit serving

### ISSUE THREAD

**Issue by heyselbi** [opendatahub-io/caikit-tgis-serving#7]

**Comment by heyselbi** [comment]
@Xaenalt could you add relevant labels and complexity etc and updates too please?

**Comment by Xaenalt** [comment]
Yep, currently working on this, also requires some debugging of the current issues with the image builds

**Comment by Xaenalt** [comment]
https://github.com/opendatahub-io/text-generation-inference/issues/8

**Comment by Xaenalt** [comment]
https://github.com/opendatahub-io/caikit-tgis-serving/pull/25

**Comment by Xaenalt** [comment]
Had a discussion with testing folks, working on moving tests into openshift-ci so we can have testing earlier in the code pipeline

**Comment by Xaenalt** [comment]
For testing, one can use `make docker-test` it'd be good to put that into CI
